src/style.css

```
@@ -1 +1,60 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
```

```suggestion
html {
```
always set font to html